ReactAndroid/src/main/java/com/facebook/react/animated/NativeAnimatedNodesManager.java

```diff
@@ -51,6 +52,9 @@
  */
 /*package*/ class NativeAnimatedNodesManager implements EventDispatcherListener {
 
+  // Used to avoid allocating a new array on every frame in runUpdates.
+  private final static List<AnimatedNode> sRunUpdateNodeList = new ArrayList<>();
```

Comment:
Yeah, I'd prefer an ivar in case someone somewhere decides to have multiple RN instances running at the same time :)